masquerade_tool/views.py

```diff
@@ -0,0 +1,68 @@
+import logging
+import boto3
+import json
+import dateutil.parser
+
+
+from datetime import datetime
+from django.conf import settings
+from django.contrib.auth.decorators import login_required
+from django.shortcuts import render
+from django.views.decorators.http import require_http_methods
+
+from django_auth_lti import const
+from django_auth_lti.decorators import lti_role_required
+from lti_permissions.decorators import lti_permission_required
+
+logger = logging.getLogger(__name__)
+AWS_REGION_NAME = settings.MASQUERADE_TOOL_SETTINGS['aws_region_name']
+FUNCTION_ARN = settings.MASQUERADE_TOOL_SETTINGS['temporary_masquerade_function_arn']
+MASQUERADE_SESSION_MINS = settings.MASQUERADE_TOOL_SETTINGS['masquerade_session_minutes']
+
+@login_required
+@lti_role_required(const.ADMINISTRATOR)
+@lti_permission_required(settings.PERMISSION_MASQUERADE_TOOL)
+@require_http_methods(['GET'])
+def index(request):
+    context = {'session_mins': MASQUERADE_SESSION_MINS}
+    return render(request, 'masquerade_tool/index.html', context)
+
+@login_required
+@lti_role_required(const.ADMINISTRATOR)
+@lti_permission_required(settings.PERMISSION_MASQUERADE_TOOL)
+@require_http_methods(['GET', 'POST'])
+def add_role(request):
+    context = {}
+    try:
+        canvas_user_id = request.LTI['custom_canvas_user_id']
+        logger.info('User {} requested masquerade access at {}'.format(
+            request.LTI['lis_person_name_full'], datetime.now()))
```

Comment:
More a question in general about our practices, but do we try to log timezone-aware dates in our log messages, vs. using e.g `utcnow()`?

Reply:
For that matter, if the log has a timestamp anyway, is this a little redundant?

Reply:
Yeah, it's redundant. I think I added it initially as it maybe  useful for  client services as they sometimes use splunk